File: app/src/main/java/com/lazylee/lzywanandroid/adapter/BaseAdapterListener.java
package com.lazylee.lzywanandroid.adapter;

import android.view.View;


public interface BaseAdapterListener {



    interface OnItemClickListener {
        void onClick(View view, int position);
    }

    interface OnItemLongClickListener{
        boolean onLongClick(View view,int position);
    }

    void setItemClickListener(OnItemClickListener itemClickListener);
    void setItemLongClickListener(OnItemLongClickListener itemLongClickListener);
}

File: app/src/main/java/com/lazylee/lzywanandroid/ui/adapter/BaseAdapterListener.kt
package com.lazylee.lzywanandroid.ui.adapter

import android.view.View


interface BaseAdapterListener {


    interface OnItemClickListener {
        fun onClick(view: View, position: Int)
    }

    interface OnItemLongClickListener {
        fun onLongClick(view: View, position: Int): Boolean
    }

    fun setItemClickListener(itemClickListener: OnItemClickListener)
    fun setItemLongClickListener(itemLongClickListener: OnItemLongClickListener)
}
